winrm4j/src/main/java/io/cloudsoft/winrm4j/winrm/WinRmTool.java

```diff
@@ -58,8 +76,23 @@ private String getEndpointUrl() {
         }
     }
 
+    /**
+     * Executes a Power Shell command.
+     * It is creating a new Shell on the destination host each time it is being called.
+     * @since 0.2
+     */
+    public WinRmToolResponse executePs(String psCommand) {
+        return executeCommand(compilePs(psCommand));
+    }
+
+    /**
+     * Execute a list of Power Shell commands as one command.
+     * The method translates the list of commands to a single String command with a <code>"\r\n"</code> delimiter and a terminating one.
+     * @deprecated since 0.2; Use the {@link #executePs(String)} instead and transform your commands list explicitly
+     */
+    @Deprecated
```

Comment:
Although I deprecated it @neykov has several tests for `executePs(List<String>)`